src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,9 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public int compareTo(Person other) {
```

Comment:
Use better variable names, 'other' is not a good name. Also compareTo is not a descriptive name